packages/main/src/BusyIndicator.js

```diff
@@ -207,7 +206,6 @@ class BusyIndicator extends UI5Element {
 		return {
 			root: {
 				"ui5-busy-indicator-root": true,
-				"ui5-busy-indicator-root--ie": isIE(),
```

Comment:
The styles, related to ui5-busy-indicator-root--ie from BusyIndicator.css  can be also removed

```css
:host([_is-busy]) :not(.ui5-busy-indicator-root--ie) ::slotted(:not([class^="ui5-busy-indicator-"])) {
	opacity: 0.6;
}

/**
 * IE fix: 0.6 makes the content almost invisible, so we set it to 0.95 in IE
 */
:host([_is-busy]) .ui5-busy-indicator-root--ie ::slotted(:not([class^="ui5-busy-indicator-"])) {
	opacity: 0.95;
}
```